### PR TITLE
Ask for multiple scopes up front, don't ask for others later.

### PR DESCRIPTION
**Pull request by jimmc**

### CODE REVIEW

File: sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts [hunk 1]
```diff
@@ -21,12 +21,14 @@ class MissingClientIdError extends Error {
 
 enum GapiScopes {
   BIGQUERY,
```
Comment (nikhilk) [inline sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts]: Could we do without needing to list BigQuery and GCS separately?

File: sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts [hunk 2]
```diff
@@ -21,12 +21,14 @@ class MissingClientIdError extends Error {
...
+  CLOUD,
   DRIVE,
   GCS,
 }
 
-// Authorize GCP scope by default
-const initialScopeString = 'profile email https://www.googleapis.com/auth/cloud-platform';
+// Ask for all necesary scopes up front so we don't need to ask again later.
+// Cloud-platform scope covers BigQuery and GCS but not Drive.
+const initialScopes = [ GapiScopes.CLOUD, GapiScopes.DRIVE ];
```
Comment (chmeyers) [inline sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts]: Do we need to include "profile" and "email" here?

Comment (jimmc) [reply sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts]: Yeah, I just noticed that, will put those back in.

File: sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts (hunk 1 above)
Comment (jimmc) [reply sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts]: Removed BIGQUERY and GCS scopes.

File: sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts (hunk 2 above)
Comment (jimmc) [reply sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts]: OK, done.

**Pull request merged**